public/index.js

```diff
@@ -1,7 +1,14 @@
-var React = window.React;
-var ReactDOM = window.ReactDOM;
+window.oc = window.oc || {};
+oc.cmd = oc.cmd || [];
```

Comment:
This may not be required. It basically is for ensuring the code will be executed either the browser oc-client being before or after the <oc-component> in a client-side or server-side rendered context.